src/api/pods.rs

```diff
@@ -495,7 +495,8 @@ impl Pods {
     /// };
     /// ```
     pub async fn prune(&self) -> Result<Vec<models::PodPruneReport>> {
-        self.podman.get_json("/libpod/pods/prune").await
+        let ep = url::construct_ep("/libpod/pods/prune", None::<&str>);
+        self.podman.post_json(ep, Payload::empty(), Headers::none()).await
```

Comment:
This can be passed directly as there is no need to append query params in this case.

```suggestion
        self.podman.post_json("/libpod/pods/prune", Payload::empty(), Headers::none()).await
```
Something like this but formatting might have to be fixed.